spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/consumer/list.html

```diff
@@ -97,6 +77,13 @@ <h2 th:text="#{admin.api-keys-consumer.subtitle}">[Post Administration</h2>
                                         <td th:text=" ${apiKey.keyValue}">Key Value</td>
                                         <td th:if="${testService == null}" class="a-center " th:text="${apiKey.created != null} ? ${#dates.format(apiKey.created, 'yyyy-MM-dd')} : ''">Created</td>
                                         <td th:if="${testService == null}" th:id="|@{item}${apiKey.id}@{ValidUntil}|" th:attr="validUntil=${#dates.format(apiKey.validUntil, 'yyyy-MM-dd')}" class="a-center " th:text="${apiKey.validUntil != null} ? ${#dates.format(apiKey.validUntil, 'yyyy-MM-dd')}: ''">Valid until</td>
+                                        <td class="right">
+                                            <button th:onclick="|@{location.href='/admin/api-keys/}${apiKey.id}@{'}|" ><i class="fa fa-search-plus"/></button>
```

Comment:
When clicking this, the details page is shown, which has a link "back to the API key list". However, that link leads to the "provider view", not the "consumer view".

Maybe the easiest solution for now would be to simply remove this link here, as the details page is anyway not really helpful.

Reply:
The details link is now deleted.